src/contracts/fund/accounting/calls/performCalculations.ts

```diff
@@ -18,10 +22,17 @@ const postProcess = async (
   prepared,
 ): Promise<PerformCalculationsResult> => {
   const quoteToken = await getQuoteToken(environment, prepared.contractAddress);
+  const hub = await getHub(environment, prepared.contractAddress);
+  const settings = await getSettings(environment, hub);
+  const fundToken = await getToken(environment, settings.sharesAddress);
+
   const calculations = {
     gav: createQuantity(quoteToken, result.gav),
     nav: createQuantity(quoteToken, result.nav),
-    sharePrice: createQuantity(quoteToken, result.sharePrice),
+    sharePrice: getPrice(
```

Comment:
hey @gempi , i don't think we need to call getPrice here; performCalculations already returns the price of the fund's share. What's the rationale?

Reply:
sorry , Simon just informed me this is token-maths getPrice, not pricefeed getPrice!